app/views/sessions/new.html.slim

```diff
@@ -0,0 +1,20 @@
+h1 class="text-secondary mt-5" = t('auth.login')
+
+= form_with(local: true, url: sessions_path) do |form|
+  = render 'sessions/shared/flash_messages'
+
+  .form-group
+    = form.label t('auth.username')
+    = form.text_field :username, required: true
+
+  .form-group
+    = form.label t('auth.password')
+    = form.password_field :password, required: true
+
+  .actions
+    = form.submit t('auth.login'), class: "btn btn-primary"
+
+  div class="small mt-3"
+    = t('app.do_not_have_account_yet')
+    | &nbsp
+    == link_to t('auth.signup'), signup_path
```

Comment:
We could also pass the `link` to I18n like this @gutakk 

```
= t('app.do_not_have_account_yet', link: link_to t('auth.signup'), signup_path)

# I18n
do_not_have_account_yet: "Content xxxx #{link}"
```

Reply:
more info here https://guides.rubyonrails.org/i18n.html#passing-variables-to-translations

Reply:
Fixed in 23e0b1dd437cda09674d878555bd9fffb308167b